reo/process_results.py

```diff
@@ -939,9 +939,13 @@ def compute_total_power(self, tech):
         if len(data['outputs']['Scenario']['Site']['PV']) == 1:
             data['outputs']['Scenario']['Site']['PV'] = data['outputs']['Scenario']['Site']['PV'][0]
 
+        
         profiler.profileEnd()
         data['outputs']["Scenario"]["Profile"]["parse_run_outputs_seconds"] = profiler.getDuration()
 
+        #Preserving Backwards Compatability
+        data['inputs']['Scenario']['Site']['LoadProfile']['outage_start_hour'] = data['inputs']['Scenario']['Site']['LoadProfile']['outage_start_time_step']
```

Comment:
@t-kwasnik outage_start_time_step = outage_start_hour + 1 (because the old inputs are zero-indexed, whereas the new inputs are meant to be more intuitive by being between 1 and 8760 for example)